spec/support.ts

```diff
@@ -18,3 +25,15 @@ export const awsMockFailureCallback = (filename) => {
         callback(response)
     }
 }
+
+// Test case data
+export const testVpc  = {
```

Comment:
hmm - this doesn't really feel like it belongs here.  But not sure where...

`VPC.spec.ts`?, or somewhere else?